portal-web/test/functional/com/liferay/portalweb/paths/portlet/openidconnect/OpenIDConnect.path

```diff
@@ -0,0 +1,38 @@
+<html>
+<head>
+<title>OpenIDConnect</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">OpenIDConnect</td></tr>
+</thead>
+
+<tbody>
+<!--ADMIN-->
+<tr>
+	<td>TOKEN_REFRESH_OFFSET</td>
```

Comment:
These names are ok, but we should think about other providers, as well distinct the fact, these are for Keycloak UI right?